CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.11.2 (2023-10-25)
+
+- Add documentation generation via yard
```

Comment:
where is this added?

Reply:
here https://github.com/ivx/ears/pull/32/files